config/locales/fr.yml

```diff
@@ -36,15 +37,18 @@ fr:
     loading: Chargement en cours...
     location: Lieu
     log_in: Se connecter
+    min_read: min lues
```

Comment:
This translation seems not reflected on:
https://github.com/forem/forem/blob/43adc9f1fbe5667340dca39cc7dcd50741d7e486/config/locales/views/articles/fr.yml#L86-L88
(might need to supply correct pluralization)